app/test/unit/git/stash-test.ts

```diff
@@ -263,20 +264,33 @@ describe('git/stash', () => {
           repository.path
         )
 
-        let status = await getStatusOrThrow(repository)
-        let files = status.workingDirectory.files
+        const status = await getStatusOrThrow(repository)
+        const files = status.workingDirectory.files
         expect(files).toHaveLength(0)
 
-        const entryToApply = entries[0]
+        entryToApply = entries[0]
         await popStashEntry(repository, entryToApply.stashSha)
+      })
 
-        status = await getStatusOrThrow(repository)
-        files = status.workingDirectory.files
+      it('restores changes and drops stash', async () => {
+        const status = await getStatusOrThrow(repository)
+        const files = status.workingDirectory.files
         expect(files).toHaveLength(1)
 
         const entriesAfter = await getDesktopStashEntries(repository)
         expect(entriesAfter).not.toContain(entryToApply)
       })
+
+      it('detects conflicted file', async () => {
```

Comment:
i think this test belongs in `status-test.ts`, but i don't feel strongly

Reply:
👍 let me sleep on this and see how I feel